package.json

```diff
@@ -68,6 +68,8 @@
   },
   "dependencies": {
     "@sendbird/chat": "^4.10.8",
+    "@sendbird/react-uikit-message-template-view": "^0.0.1-alpha.61",
+    "@sendbird/uikit-message-template": "^0.0.1-alpha.61",
```

Comment:
```suggestion
    "@sendbird/react-uikit-message-template-view": "0.0.1-alpha.61",
    "@sendbird/uikit-message-template": "0.0.1-alpha.61",
```

Reply:
bc future might contain breaking change?

Reply:
I'm not sure, but this is to prevent unintended version modifications.